server/routes/api/disaster_api.py

```diff
@@ -74,5 +75,16 @@ def event_data():
   place = request.args.get('place', '')
   if not place:
     return "error: must provide a place field", 400
-  result = dc.get_event_collection(event_type, place, date)
+  filter_prop = request.args.get('filterProp', '')
+  filter_unit = request.args.get('filterUnit', '')
+  filter_upper_limit = request.args.get('filterUpperLimit', None)
+  filter_lower_limit = request.args.get('filterLowerLimit', None)
+  # if filtering values, need all of filter prop, filter upper limit, and filter
+  # lower limit. Filter unit is optional.
+  if filter_prop and filter_upper_limit and filter_lower_limit:
+    result = dc.get_event_collection(event_type, place, date, filter_prop,
+                                     filter_unit, float(filter_upper_limit),
+                                     float(filter_lower_limit))
+  else:
+    result = dc.get_event_collection(event_type, place, date)
```

Comment:
is this else condition necessary? If the filter params are not set (and take 0 value), there will be no filtering in mixer.

Reply:
if we get an empty filterUpperLimit or empty filterLowerLimit from the client, converting a None value to a float will throw an error so I thought it would be easier to handle it this way

Reply:
Maybe do `filter_upper_limit = request.args.get('filterUpperLimit') or 0`, then can just keep line 85-87 with full args

Reply:
done